core/embed/sec/tropic/tropic.c

```diff
@@ -47,13 +56,20 @@ bool tropic_init(void) {
     return true;
   }
 
-  curve25519_key tropic_pubkey = {0};
-  curve25519_key trezor_privkey = {0};
+#ifdef TREZOR_EMULATOR
+  drv->device.addr = inet_addr("127.0.0.1");
+  drv->device.port = 28992;
+  drv->device.rng_seed = (unsigned int)time(NULL);
```

Comment:
I think you don't need to set `rng_seed`. It happens in [here](https://github.com/tropicsquare/libtropic/blob/develop/hal/port/unix/lt_port_unix_tcp.c#L211).

Reply:
removed https://github.com/trezor/trezor-firmware/commit/36cb2f23a8910535b78b2db06369f0b59e1a3cb3